jscript/jquery.js

```diff
@@ -0,0 +1,40 @@
+$(document).ready(function() {
+
+	$('.read-more').click(showMore);
+	$('.learn-more').click(learnMore);
+
```

Comment:
The instructions also call for a `click` event to wired up when the user clicks on the `.read-less` link. This should effectively undo what you did in your `showMore()` click handler function.